lib/node_modules/@stdlib/array/complex64/README.md

```diff
@@ -1478,6 +1477,26 @@ idx = arr.indexOf( new Complex64( 1.0, -1.0 ), 1 );
 // returns -1
 ```
 
+<a name="method-join"></a>
+
+#### Complex64Array.prototype.join( \[ separator ] )
+
+Returns string representations of array elements.
+
+```javascript
+var arr = new Complex64Array( 3 );
+
+arr.set( [ 1.0, 1.0 ], 0 );
+arr.set( [ 2.0, -2.0 ], 1 );
+arr.set( [ 3.0, 3.0 ], 2 );
+
+var str = arr.join( '/' );
+// returns '1 + 1i/2 - 2i/3 + 3i'
+
+str = arr.join();
+// returns '1 + 1i,2 - 2i,3 + 3i'
```

Comment:
```suggestion
var str = arr.join();
// returns '1 + 1i,2 - 2i,3 + 3i'
```

By default, the method separates serialized array elements by a comma. To use an alternative separator, provide a `separator` string.

```javascript
var arr = new Complex64Array( 3 );

arr.set( [ 1.0, 1.0 ], 0 );
arr.set( [ 2.0, -2.0 ], 1 );
arr.set( [ 3.0, 3.0 ], 2 );

var str = arr.join( '/' );
// returns '1 + 1i/2 - 2i/3 + 3i'
```